drafts/tests/test__apicli_via_bash_powered.tasks.yml

```diff
@@ -0,0 +1,61 @@
+# https://taskfile.dev
+
+version: '3'
+
+vars:
+  tool_path: python impls/reverser_impl_sam.py
+#  tool_path: |-
+#    python -c 'import sys; import os; print("" if len(sys.argv) < 2 else str(sys.argv[0] or ""))[::-1]'
+
+tasks:
+  default:
+    cmds:
+      - echo "{{.GREETING}}"
+    silent: true
+
+  test_on_op_full__cmdlines:
+    desc: run test list
+    cmds:
+    - task test_apipy:check-only-letters-more-than-5
+    - task test_apipy:check-letters-with-numbers-even
+    - task test_apipy:check-letters-single
+
+  check-only-letters-more-than-5: |-
+    task test_apipy:test_one_op_jscmd title="{{.TASK}}" \
+      _item='{send_input: abcdef, expect_output: fedcba}'
+
+  check-letters-with-numbers-even: |-
```

Comment:
I suggest to rename the test to `check-string-with-even-number-of-letters`

Reply:
fixed